packages/dcos-ui/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-ui.git",
-    "ref": "b51934df009bd2a86dec605cdc5b34f006feffef",
-    "ref_origin": "master"
+    "ref": "df3fc757e36684c307b55f6baf2085c5fa0120fb",
+    "ref_origin": "release/1.9"
```

Comment:
This seems to be going back a considerable number of commits (https://github.com/dcos/dcos-ui/compare/df3fc757e36684c307b55f6baf2085c5fa0120fb...b51934df009bd2a86dec605cdc5b34f006feffef). Is that intentional?